### ISSUE TITLE
Keyboard Bindings

### ISSUE THREAD

**Issue by vvendelbo** [hrjakobsen/slext#70]
The keyboard binding for alt + 8 and alt + 9 for changing tabs are the same as for "[" and "]".
Possibility to reassign the hot keys would be helpful.

**Comment by hrjakobsen** [comment]
Is this on a Mac?

**Comment by hrjakobsen** [comment]
Also - related to #60

**Comment by vvendelbo** [comment]
Yup

**Comment by hrjakobsen** [comment]
I've implemented a custom meta-key in 2fc017e934bd8db669e5cfe49c267a381bc1ad11, so that it can be changed to one of the following combinations:

- Alt
- Alt + Ctrl
- Ctrl 
- Cmd/Win
- Cmd/Win + Alt
- Cmd/Win + Ctrl

But it needs some more testing for me to push this onto the Chrome Webstore. If you would like to help test it, you can checkout the branch `feature-custom-keybindings` and compile SLext.

**Comment by Bargsteen** [comment]
I've tested the shortcuts on MacOS 10.15 with Chrome and Brave. Currently, the best solution is to use `Ctrl + Alt` as the meta key and then using `Alt + G` for jumping to files (which works without changing settings).

I've created a spreadsheet deliberating what does and does not work including the side-effects.
https://docs.google.com/spreadsheets/d/1bnybEFazkekRNiTIniVq_yk7c3UAOpe2KN-Oi40wlZo/edit#gid=0
